PowerShell Scanners/Windows Defender Threats/Windows Defender Threat Detection.ps1

```diff
@@ -0,0 +1,11 @@
+﻿if (-Not(Get-Command Get-MpComputerStatus -ErrorAction SilentlyContinue)) {
+    throw "Unable to find Get-MpComputerStatus. Available on Windows 10/Server 2016 or higher"

```

Comment:
Get-MpComputerStatus needs to be changed to Get-MpThreatDetection.

Reply:
Copy paste failure fixed.